Client/Extensions/AppearanceExtensions.swift

```diff
@@ -72,8 +73,13 @@ extension Theme {
             UIView.appearance().appearanceOverrideUserInterfaceStyle = isDark ? .dark : .light
         } else {
             // iOS 12 fixes, many styling items do not work properly in iOS 12
-            UILabel.appearance().appearanceTextColor = colors.tints.home
+            let views = [ShieldsViewController.self, SyncViewController.self,
```

Comment:
This isn't ideal but the only way to work around of setting global appearance.

Alternatively I could create a custom `ThemableViewController` and use it for these VCs.
I didn't decide to do that because this is a temporary solution until we move rewards-ui to brave-ios and decide how to handle dark mode theming.